docs/querying/sql-functions.md

```diff
@@ -133,6 +134,30 @@ Returns any value of the specified expression.
 * **Syntax**: `ANY_VALUE(expr, [maxBytesPerValue, [aggregateMultipleValues]])`
 * **Function type:** Aggregation
 
+<details><summary>Example</summary>
+
+The following example returns the state abbrevation, state name, and average flight time grouped by each state in `flight-carriers`:
+
+```sql
+SELECT
+  "OriginState",
+  ANY_VALUE("OriginStateName") AS "OriginStateName",
```

Comment:
Since I would expect all the values for `OriginStateName` to be the same, wonder if another example might be a smidge more illustrative. For example:

```
SELECT
  "Reporting_Airline",
  ANY_VALUE("OriginStateName") AS "OriginStateName"
--  AVG("ActualElapsedTime") AS "AverageFlightTime"
FROM "flight--carriers"
GROUP BY 1
LIMIT 3
```
Commented out average for this one since it is irrelevant.

Reply:
I looked at various examples of ANY_VALUE and it looks like it's often used to correlate a group by field (originstate in this example) with a similar identifier that's 1:1. So that you can get additional information on the field grouped by, but you don't have to do `GROUP BY 1, 2`, potentially for query optimization reasons

this was one of the references I used: https://docs.oracle.com/en/database/oracle/oracle-database/21/sqlrf/ANY_VALUE.html